vendor/golang.org/x/text/encoding/charmap/maketables.go

```diff
@@ -0,0 +1,556 @@
+// Copyright 2013 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+package main
+
```

Comment:
Hmm, what's this dependency for? Just curious.

Reply:
That just comes in as a transitive dependency of ginkgo and/or gomega. Should be in your version test PR, too.